examples/native-fungible/src/service.rs

```diff
@@ -61,8 +62,27 @@ impl Accounts {
             AccountOwner::Application(_) => return Err(Error::ApplicationsNotSupported),
         };
 
-        let balance = system_api::current_owner_balance(owner);
-        Ok(AccountEntry { value: balance })
+        Ok(AccountEntry {
+            key,
+            value: system_api::current_owner_balance(owner),
+        })
+    }
+
+    async fn entries(&self) -> Result<Vec<AccountEntry>, Error> {
+        Ok(system_api::current_owner_balances()
+            .into_iter()
+            .map(|(owner, amount)| AccountEntry {
+                key: AccountOwner::User(owner),
+                value: amount,
+            })
+            .collect())
+    }
+
+    async fn keys(&self) -> Result<Vec<AccountOwner>, Error> {
+        Ok(system_api::current_owners()
```

Comment:
Maybe we should call it `current_balance_owners`, to distinguish it from the _chain_ owners.